Homework 5-6

```diff
@@ -0,0 +1,98 @@
+HOMEWORK WEEK 5-6 
+(handout for students) 
+TASK 1 (Agile Techniques) 
+Question 1
+Complete definitions for Scrum related key terminology provided below. 
+SCRUM CEREMONIES 
+·  Product backlog refinement is a type of meeting held toward the end of the sprint to check that the backlog has been achieved before the next one
+•	Sprint planning, this is when you prioritise the tasks in a sprint and plan the order they are going to be actioned
+•	  Daily scrum , this is held at the start of the day to see the progress in a sprint held by a scrum master.
```

Comment:
It is key to mention that this is a short daily meeting (typically 15 minutes) setting the context for the day, where team members are asked what they did yesterday, what they are going to do today and whether there are any impediments in their way.